### DISCUSSION TITLE
Is it possible to add hyperlinks to timeline diagram?

### DISCUSSION THREAD

**Discussion by crimsonvspurple** [mermaid-js/mermaid#4205] [Q&A]
I am new to mermaid and tried to add links (md and html) on timeline diagram items but it didn't work. any idea?

I looked here https://github.com/mermaid-js/mermaid/tree/develop/packages/mermaid/src/diagrams/timeline but not seeing anything related to links.

**Comment by cobyiv** [comment]
I second this issue. would be great to have.

**Comment by indexing** [comment]
I submitted an issue requesting this functionality: https://github.com/mermaid-js/mermaid/issues/5249
